.github/workflows/reusable_run_tox_test.yml

```
@@ -64,8 +64,14 @@ jobs:
       COVERAGE: ${{ inputs.coverage }}
       TOX_WORK_DIR: .tox
       TOX_EXTRAS: ${{ inputs.tox_extras }}
+      POOCH_HOME: ${{ github.workspace }}/.pooch
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      - name: Cache pooch downloads
+        uses: actions/cache@0400d5f644dc74513175e3cd8d07132dd4860809 # v4
+        with:
+          path: ${{ env.POOCH_HOME }}
+          key: pooch-v
```

```suggestion
          key: pooch-v1
```
?

But, why are we using a versioned key?

---

A need to manually invalidate the cache.
But Pooch already has hashes...